<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogstraightline</class>
 <widget class="QDialog" name="Dialogstraightline">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Straightline</string>
  </property>
  <widget class="QGroupBox" name="GB_setI">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>startpoint --- I vessel</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V (m/s)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="LE_setxI0"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setyI0"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setvelocityI"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Button --- I vessel</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="setupsetpointI">
       <property name="text">
        <string>setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetsetpointI">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_setI_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Endpoint --- I vessel</string>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>theta (deg)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLineEdit" name="LE_setxI1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setyI1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setthetaI1"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_setI_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>startpoint --- II vessel</string>
   </property>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V (m/s)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="LE_setxII0"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setyII0"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setvelocityII"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GB_setI_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>190</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Endpoint --- II vessel</string>
   </property>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>x (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y (m)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>theta (deg)</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <widget class="QLineEdit" name="LE_setxII1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setyII1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_setthetaII1"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>211</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Button --- II vessel</string>
   </property>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QPushButton" name="setupsetpointII">
       <property name="text">
        <string>setup</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetsetpointII">
       <property name="text">
        <string>reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
